index.html

```diff
@@ -85,30 +99,33 @@ <h2 class="fw-normal">I am Hamza</h2>
     <!-- Projects section -->
     <section class="projects">
       <div class="projects-container">
-        <nav class="projects-title d-flex flex-dir-col">
+        <div class="projects-title d-flex flex-dir-col">
           <h2 class="pro-title">My Recent Works</h2>
-          <div class="underline"></div>
-        </nav>
+          <span class="underline"></span>
+        </div>
         <div class="multi-poste d-flex flex-dir-col">
           <img
             src="./image_icons/Img_Placeholder.svg"
             alt="image_Placeholder"
           />
-          <h3 class="intru-title">Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="languages">
-            <li><p>css</p></li>
-            <li><p>html</p></li>
-            <li><p>bootsrap</p></li>
-            <li><p>Ruby</p></li>
-          </ul>
-          <button class="btn-primary btn-project" type="button">
-            See Project
-          </button>
+          <article class="d-flex flex-dir-col">
+            <h3 class="intru-title">Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages">
+              <li><p>css</p></li>
+              <li><p>html</p></li>
+              <li><p>bootsrap</p></li>
+              <li><p>Ruby</p></li>
+            </ul>
```

Comment:
- [ ] This is not matching with the origianl design. Please work on it. 

**Clone** 

<img width="437" alt="Capture" src="https://user-images.githubusercontent.com/58504655/125781403-2c61e378-e579-4611-b791-e2915c1d2bd9.PNG">